src/main/java/igrad/commons/core/Messages.java

```diff
@@ -9,6 +9,7 @@
     public static final String MESSAGE_UNKNOWN_COURSE_COMMAND = "Unknown course command, possible options;\n"
         + "course add ...\n"
         + "course edit ...\n"
+        + "course assign ...\n"
```

Comment:
no Teri, this should not be there :)

Reply:
its only for `requirement assign`

Reply:
oops, I wrote wrongly, wanted to write Achieve

Reply:
oh ok